test/unit/view.child-views.spec.js

```diff
@@ -300,7 +300,7 @@ describe('layoutView', function() {
       var ChildView = Marionette.View.extend({
         template: false,
         onRender: function() {
-          this.triggerMethod('content:rendered');
```

Comment:
Is this signalling a move towards removing `triggerMethod` in favour of just using `trigger` everywhere instead?